tasks/docker-offline-inst.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Create the directory where rpm will be copied
+  file:
+    path: "{{ docker_dest_dir }}"
+    state: directory
+    mode: '0755'
+
+- name: Copy docker tar on target machine
+  copy:
+      src: files/docker.tar
+      dest: "{{ docker_dest_dir }}"
+
+- name: unpack docker tar
+  shell: tar -xvf docker.tar
+  args:
+    chdir: "{{ docker_dest_dir }}"
```

Comment:
Why are we assuming an uncompressed tar? Why are we assuming the source file location? Why use shell module instead of command or unarchive?

Reply:
I assume that tar file is a quick way to transfer several files at once, tar has been placed under files dir cause is an artifact used by task (see also the files about docker compliance, like docker.json).

For shell command, I'll try to avoid as much as possible and use the command or ansible modules.